### ISSUE TITLE
Improve requirePaddingNewLinesInObjects

### ISSUE THREAD

**Issue by yekver** [jscs-dev/node-jscs#1551]
We can ignore object without padding on new lines if there is less that `N` options in this object.
For example:
`"requirePaddingNewLinesInObjects": 1`

ok:
```javascript
var a = {a: 1};
var a = {
      a: 1,
      b: 2
};
```
fail:
```javascript
var a = {a: 1, b: 2};
```

**Comment by mrjoelkemp** [comment]
Hi @yekver. Thanks for contributing. So that I'm clear, you're proposing a new rule value for the `requirePaddingNewLinesInObjects` rule?

**Comment by yekver** [comment]
@mrjoelkemp Yep, you understand me correctly!
The point of my advice is that there is no reason to require padding new lines in object that contains only one element. For two or more elements in object padding new lines are required, because they make `git diff` looks prettier for watching elements that was deleted, updated or inserted.

**Comment by yekver** [comment]
Yeah! I think this would be great.

P.S. For implementing this case we need separated rule?
ok:
```javascript
var a = {
      a: 1
},
{
      a: 1,
      b: 2
};
```
fail:
```javascript
var a = {
      a: 1
}, {
      a: 1,
      b: 2
};
```

**Comment by markelog** [comment]
@yekver would you be interested in implementing that feature?

**Comment by hzoo** [comment]
We released [3.0](https://github.com/jscs-dev/node-jscs/releases/tag/v3.0.0) recently! We also wrote a [post](https://medium.com/@markelog/jscs-end-of-the-line-bc9bf0b3fdb2#.oot3wvgpu) about how 3.0 will be our last major release (since we are deprecating the project) and that we are [merging with ESLint](http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint)!

Thus we won't be working on any new features moving forward which includes new rules/options. We will continue to fix bugs and work on CST/better autofixing for current rules in the upcoming months both to help JSCS users and to gain experience that we can hopefully use in ESLint. But otherwise we both teams will be working on getting [ESLint feature parity with JSCS](https://github.com/eslint/eslint/milestones/JSCS%20Compatibility) and just a nice transition for our users.

I'm putting an `orphaned` label to note this issue isn't planned to be worked on, but might be a good contribution as a custom rule (not in JSCS core itself).

I would encourage anyone who wants the rule to try writing it themselves: I would either write a custom rule or submit an issue to ESLint and if accepted a PR. Otherwise a custom rule that won't be in core repo itself.

JSCS rules are also very similar to ESLint rules: here are some resources to get started: 

- [Project-specific Lint Rules with ESLint](https://gist.github.com/jareware/7179093)
- [Creating Custom Rules for ESLint](https://insideops.wordpress.com/2015/12/08/creating-custom-rules-for-eslint)
- [Maintaining JavaScript Code Quality with ESLint ](https://www.paypal-engineering.com/2014/12/12/maintaining-javascript-code-quality-with-eslint/)
